pkg/rulerequest/handle.go

```diff
@@ -24,6 +26,18 @@ func PromptRun(ctx context.Context, c client.Client, dangerous bool, image strin
 			app, err = c.AppRun(ctx, image, &opts)
 		}
 	}
+	if naErr := (*imageallowrules.ErrImageNotAllowed)(nil); errors.As(err, &naErr) {
+		err.(*imageallowrules.ErrImageNotAllowed).Image = image
+		if choice, promptErr := handleNotAllowed(dangerous, image); promptErr != nil {
+			return nil, fmt.Errorf("%s: %w", promptErr.Error(), err)
+		} else if choice != "NO" {
```

Comment:
Based on the choice map that gets created in `handleNotAllowed`, I think it would return lowercase `"no"` and not all caps. Might be best to just always use all caps (there and in this function) to keep it consistent.

Reply:
true!